tests/fv/helper/calico_version_helper.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/docopt/docopt-go"
+	clientv3 "github.com/projectcalico/libcalico-go/lib/clientv3"
+
+	"github.com/projectcalico/calicoctl/v3/calicoctl/commands/clientmgr"
+	"github.com/projectcalico/calicoctl/v3/calicoctl/commands/constants"
+	"github.com/projectcalico/calicoctl/v3/calicoctl/commands/datastore/migrate"
+)
+
+var VERSION string
+
+func main() {
+	doc := `Usage:
+  calico_version_helper [options]
+
+Options:
+  -h --help                 Show this screen.
+  -v --version=<version>    Version to set.
+                            [default: ` + VERSION + `]
+  -c --config=<config>      Path to the file containing connection
+                            configuration in YAML or JSON format.
+                            [default: ` + constants.DefaultConfigPath + `]
+  --context=<context>       The name of the kubeconfig context to use.
+
+Description:
+  Set CalicoVersion in ClusterInformation.
+`
+	parsedArgs, err := docopt.ParseDoc(doc)
+	if err != nil {
+		fmt.Printf("Could not parse arguments: %s, err: %v\n", strings.Join(os.Args[1:], " "), err)
+		os.Exit(1)
+	}
+
+	if context := parsedArgs["--context"]; context != nil {
+		os.Setenv("K8S_CURRENT_CONTEXT", context.(string))
+	}
+
+	cf, _ := parsedArgs["--config"].(string)
+
+	cfg, err := clientmgr.LoadClientConfig(cf)
+	if err != nil {
+		fmt.Printf("Could not load client config: %v\n", err)
+		os.Exit(1)
+	}
+
+	// Get the backend client for updating cluster info and migrating IPAM.
+	client, err := clientv3.New(*cfg)
+	if err != nil {
+		fmt.Printf("Could not create client: %v\n", err)
+		os.Exit(1)
+	}
+
+	// Import Calico CRDs
+	err = migrate.ImportCRDs(cfg)
```

Comment:
Do we need to do this for etcd clusters as well? I figured that the CRDs would only be needed for KDD clusters.

Reply:
I **think** we could also apply just the `ClusterInformation` CRD we need in the K8s test cluster we're using too via `kubectl` (or some other method of hitting the K8s API)? Then we could avoid needing a blanket import as well.

Reply:
hm... I just figured we already had a way of updating the CRDs when they change, and since "utility" app is only used on tests, maybe this was fine... I'll look into only importing the `ClusterInformation` CRD

Reply:
as we sync'ed up offline, I'll import the CRDs only on kdd via the Makefile for the test environments, so we won't need to worry about making `importCRDs()` public in `migrate`, or about importing them in the go code. Thanks for helping me out with this!